### PR TITLE
chore : Lint fix 

### PR DESCRIPTION
**Pull request by neeldoshii**
Fixes
- Using 'Log' instead of 'Timber'
- Line is longer than allowed by code style (> 100 columns)
- Use of getter method instead of property access syntax
- Should be replaced with Kotlin function
- Cascade 'if' should be replaced with 'when'

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/edit/EditActivity.kt [hunk 1]
```diff
@@ -42,8 +43,11 @@ class EditActivity : AppCompatActivity() {
         supportActionBar?.title = ""
         val intent = intent
         imageUri = intent.getStringExtra("image") ?: ""
-        vm = ViewModelProvider(this).get(EditViewModel::class.java)
+        vm = ViewModelProvider(this)[EditViewModel::class.java]
         val sourceExif = imageUri.toUri().path?.let { ExifInterface(it) }
+        // TODO(Deprecation : 'TAG_APERTURE: String' is deprecated. Deprecated in Java)
```
Comment (neeldoshii) [inline app/src/main/java/fr/free/nrw/commons/edit/EditActivity.kt:48]: For Maintainers : should we create separate issue for this referencing deprecation for it and comment issue here?

Comment (nicolas-raoul) [reply app/src/main/java/fr/free/nrw/commons/edit/EditActivity.kt:48]: Yes thanks! 🙂

**Review by nicolas-raoul — APPROVED** [review]
Tested with a picture I had to rotate: https://commons.wikimedia.org/wiki/File:Fancy_dresses_in_Paris.jpg
Worked well.
Thanks! :-)

**Review by misaochan — APPROVED** [review]

**Comment by misaochan** [comment]
Thanks @neeldoshii !

**Pull request merged**